rust/template/differential_datalog/Cargo.toml

```diff
@@ -16,6 +16,7 @@ ordered-float = {version = "2.0.0", features = ["serde"]}
 fnv = "1.0.2"
 timely = "0.11"
 libc = "0.2"
+csv = "1.1"
```

Comment:
I don't know how you feel about adding more dependencies to the project? We can also consider other file formats if CSV seems to verbose or slow.

Reply:
I've no problem with adding this dependency.